ARM-Samples/Create-Video-Analyzer/README.md

```diff
@@ -0,0 +1,92 @@
+
+# Overview
+
+In this Quick-Start you will create an Azure Analyzer for Media (a.k.a "AVAM") resource by using Arm Template (PREVIEW)
+The resource will be deployed to your subscription and will create the 'AVAM' resource based on parametrers presented on the avam.template file.
+
+> **Note:**
+> this sample is *not* for migrating an existing AVAM account to an ARM-Based AVAM account.
+> For a full documentation on AVAM API, visit the [AVAM Developer Portal](https://aka.ms/avam-dev-portal) page.
+> The current API Version is "2021-08-16-preview". Check this Repo from time to time to get updates on new API Versions.

```

Comment:
We already have 2021-09-01-preview up and running

Reply:
is there something changed ? This is why I've asked to see an Template sample.

Reply:
yes things changed with the backend of ARM, RPaaS and AVAM between the versions. The template schema is the same, but the request must come in with the new version for it to work properly in the backend.